x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/count.tsx

```diff
@@ -128,6 +128,7 @@ export const countOperation: OperationDefinition<CountIndexPatternColumn, 'field
       isBucketed: false,
       scale: 'ratio',
       sourceField: field.name,
+      timeScalingMode: 'optional',
```

Comment:
The time scaling mode is defined on the operation, no need to define it on each individual column. We also don't do this in other places (and you are not reading the time scaling mode from the column anywhere, only from the operation definition)

Reply:
good point!